data/adapters/STARR_seq_adapter.py

```diff
@@ -0,0 +1,177 @@
+from concurrent.futures import ThreadPoolExecutor
+
+from biocommons.seqrepo import SeqRepo
+from ga4gh.vrs.extras.translator import AlleleTranslator
+from ga4gh.vrs.dataproxy import SeqRepoDataProxy
+import hashlib
+import csv
+import json
+from adapters.helpers import bulk_check_spdis_in_arangodb, load_variant
+from adapters.file_fileset_adapter import FileFileSet
+from typing import Optional
+
+from adapters.writer import Writer
+
+# Example rows from Gersbach's STARR-seq data
+# chrom	chromStart	chromEnd	name	score	strand	log2FoldChange	inputCountRef	outputCountRef	inputCountAlt	outputCountAlt	minusLog10PValue	minusLog10QValue	postProbEffect	CI_lower_95	CI_upper_95	variantPos	refAllele	altAllele
+# chr1	13833	13834	NC_000001.11:13833:C:T	350	+	0.1053361347394244	1.1178449514319324	1.0921171037854174	0	0.2149853195124717			0.35	0.620196	4.98262	-1	C	T
+# chr1	14264	14265	NC_000001.11:14264:C:T	196	+	-0.001248471	0.5788839927058221	0.8904954846250326	0	0.085994128			0.196	0.420745	2.70963	-1	C	T
+# chr1	14470	14471	NC_000001.11:14470:G:A	221	+	-0.030633972	0.7984606795942375	0.8232882782382377	0.1013695533505425	0.085994128			0.221	0.280078	1.73764	-1	G	A
+# chr1	14792	14793	NC_000001.11:14792:G:A	285	+	0.051253758	0.8783067475536612	0.5376576510943594	0	0.085994128			0.285	0.538601	5.691	-1	G	A
+# chr1	15189	15190	NC_000001.11:15189:C:T	174	+	-0.00039391	0.6387685436753899	0.7056756670613465	0.050684777	0.128991192			0.174	0.454782	1.97316	-1	C	T
+# chr1	15777	15778	NC_000001.11:15777:A:G	139	+	0.018235348	0.4391533737768306	0.5376576510943594	0	0.085994128			0.139	0.543171	2.39533	-1	A	G
+# chr1	16101	16102	NC_000001.11:16101:T:G	183	+	-0.012858891	0.3992303397971187	0.5712612542877568	0	0.085994128			0.183	0.489794	1.80506	-1	T	G
+# chr1	16280	16281	NC_000001.11:16280:T:C	165	+	-0.002886834	0.4391533737768306	0.7896846750448403	0.050684777	0.2149853195124717			0.165	0.516841	2.67298	-1	T	C
+# chr1	16949	16950	NC_000001.11:16949:A:C	192	+	-0.016602482	1.1777295024015002	0.5040540479009619	0.60821732	0.2579823834149661			0.192	0.573042	1.67965	-1	A	C
+# chr1	17005	17006	NC_000001.11:17005:A:G	299	+	0.094992695	1.7765750120971782	0.6048648574811543	0.050684777	0.128991192			0.299	0.544042	3.77914	-1	A	G
+# chr1	17020	17021	NC_000001.11:17020:G:A	209	+	-0.017787266	1.6168828761783307	0.5712612542877568	0.8616412034796114	0.2579823834149661			0.209	0.513392	1.55285	-1	G	A
+# chr1	17222	17223	NC_000001.11:17222:A:G	457	+	-0.266156311	1.317460121330492	0.554459453	1.926021513660308	0.3869735751224492			0.457	0.357333	1.17642	-1	A	G
+# chr1	17385	17386	NC_000001.11:17385:G:A	222	+	0.014391003	1.6168828761783307	0.5880630558844554	0	0.042997064			0.222	0.45808	3.30742	-1	G	A
+
+
+class STARRseqVariantOntologyTerm:
+    ALLOWED_LABELS = ['variant', 'variant_ontology_term']
```

Comment:
usually call those collections variants_biosamples or variants_phenotypes, as more specific to ontology_terms

Reply:
fixed.